app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/LearningActivity.kt

```diff
@@ -21,11 +22,27 @@ class LearningActivity : AppCompatActivity() {
 
     private val dummyPres = DummyUIPresenter(DummyDatabaseService())
     private val learningPresenter = LearningPresenter(DummyDatabaseService())
+    private lateinit var audioFeedback : LearningAudioFeedback
```

Comment:
Does it really need to be a lateninit? (could be changed for next sprint)

Reply:
I think it does, because I need the applicationContext, but I need to check